### DISCUSSION TITLE
Is there some way to change the default font-weight of n-strong?

### DISCUSSION THREAD

**Discussion by mitya33** [tusen-ai/naive-ui#5114] [Q&A] [answered]
It seems to use 500, and there is no variable in the typography component (nor prop) to override this. In many fonts, 500 looks barely different from 400. Bold/strong should begin at 600, I'd say.

**Comment by sahitono** [comment]
have you tried customize theme like this https://www.naiveui.com/en-US/os-theme/docs/customize-theme ? then change the `fontWeightStrong` field

**Comment by mitya33** [comment]
Thank you - I've just realised you mean within the `Common` theme overrides. I was looking (and failing to find) a variable to override within the `Typography` component, which doesn't provide such a variable.

**Comment by Volankey** [comment, marked as answer]
```
 <n-text
    :theme-overrides="{
      common: {
        fontWeightStrong: 900,
      },
    }"
    strong
    type="info"
  >
    Info
  </n-text>
```

**Comment by PANstudio** [comment]
Thanks. Out of curiosity, why is this a `common` variable and not a `typography` one?
